core/server/api/app.js

```diff
@@ -170,17 +170,21 @@ function apiRoutes() {
 
     // ## Authentication
     apiRouter.post('/authentication/passwordreset',
-        spamPrevention.forgotten,
+        // Prevent more than 5 password resets from an ip in an hour for any email address
+        brute.globalReset,
+        // Prevent more than 5 password resets in an hour for an email+IP pair
+        brute.userReset,
         api.http(api.authentication.generateResetToken)
     );
-    apiRouter.put('/authentication/passwordreset', api.http(api.authentication.resetPassword));
+    apiRouter.put('/authentication/passwordreset', brute.globalBlock, api.http(api.authentication.resetPassword));
```

Comment:
I have a question: I've added this TODO in one of my last tidy up PR's, see https://github.com/TryGhost/Ghost/pull/7579/files#diff-25f0b2b0cc338d73a55da195594de403R307

Is this TODO true?Is it possible to use the new brute middleware for the token locked logic?
Thanks